spark/resources/install.txt

```diff
@@ -0,0 +1,15 @@
+
+#Jupyter notebook installation for Mac
```

Comment:
If it is there I didn't see it but are we pointing to this file from the readme or etc?

Reply:
Its funny, even I can't see it, in browser. Added a reference to install.txt in readme.

Reply:
Now it is visible.